src/modules/weather/weather.module.js

```diff
@@ -0,0 +1,199 @@
+import api from './weather.api';
+
+export class Weather {
+    constructor() {
+        this.viewElement = document.querySelector('#main');
+        this._start();
+    }
+
+    _start() {
+        this._fiveDayWeather = api.getFiveDayWeather();
+        this._currentDayWeather = api.getCurrentDayWeather();
+        this._currentLocationInfo = api.getCurrentLocationInfo();
+        this._createContainer();
+    }
+
+    _createContainer() {
+        this.container = document.createElement('div');
+        this.viewElement.appendChild(this.container);
+        this._createToday();
+        this._createNextDays();
+    }
+
+    _createToday() {
+        const currentDayDiv = createRow();
+        const fiveDayWeather = this._fiveDayWeather;
+        const currentDayWeather = this._currentDayWeather;
+        const currentLocationInfo = this._currentLocationInfo;
+        const createWeatherElement = this._createWeatherElementIn;
+        createCurrentDayInfoIn(currentDayDiv);
+        createCurrentDayWeatherIn(currentDayDiv);
+        this.container.append(currentDayDiv);
+
+        function createCurrentDayInfoIn(parent) {
```

Comment:
I would not recomend to nest functions inside. Its better to add methods to parent class instead of putting in methods - better approach for testing and more readable